From 5844c083c2bd5c49da67b5e30e2f7f180c32d175 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:08:30 -0400
Subject: [tools] perf: Fix "==" into "=" in ui_browser__warning assignment

Message-id: <1410181752-25631-266-git-send-email-jolsa@redhat.com>
Patchwork-id: 91885
O-Subject: [PATCH RHEL7.1 BZ1134356 265/307] perf tools: Fix "==" into "=" in ui_browser__warning assignment
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 7f3e508ee1e6cc1b5865edcbf04a14a76ff1534c
Author: zhangdianfang <zhangdianfang@huawei.com>
Date: Fri May 30 08:53:58 2014 +0800

description
===========
Convert "==" into "=" in ui_browser__warning assignment.

Bug description: https://bugzilla.kernel.org/show_bug.cgi?id=76751

[ changed the changelog a bit ]
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/ui/browser.c b/tools/perf/ui/browser.c
index d11541d..3ccf6e1 100644
--- a/tools/perf/ui/browser.c
+++ b/tools/perf/ui/browser.c
@@ -194,7 +194,7 @@ int ui_browser__warning(struct ui_browser *browser, int timeout,
   ui_helpline__vpush(format, args);
   va_end(args);
  } else {
-  while ((key == ui__question_window("Warning!", text,
+  while ((key = ui__question_window("Warning!", text,
          "Press any key...",
          timeout)) == K_RESIZE)
    ui_browser__handle_resize(browser);
-- 
1.7.1